translations/translations.json

```diff
@@ -3110,7 +3121,18 @@
       "today": "today",
       "todayRecurring": "today, recurring",
       "onlyThisEvent": "Only this event",
-      "thisAndAllFutureEvents": "This and all future events"
+      "thisAndAllFutureEvents": "This and all future events",
+      "from": "from",
+      "to": "to",
+      "resetFilter": "reset filter:",
+      "invalid_from_date": "Invalid from date format",
+      "invalid_to_date": "Invalid to date format",
+      "should_be_earlier": "from date should be earlier than to date",
+      "hide_filters": "Hide filters",
+      "show_filters": "Show filters",
+      "select_range": "Select Range",
+      "ok": "Ok",
+      "cancel": "Cancel"
```

Comment:
`cancel` should be available in the `layout` namespace already. Since `ok` is also very generic, you could just add it there too and use those translations. You can do so like this:
``` ts
t('layout:cancel')
```

Reply:
That's what I was looking for Thanks!